doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -361,6 +361,8 @@ nRF Desktop
   * Bluetooth connectivity support (:ref:`CONFIG_DESKTOP_BT <config_desktop_app_options>`) implies using a separate workqueue for connection TX notify processing (:kconfig:option:`CONFIG_BT_CONN_TX_NOTIFY_WQ`) if MPSL is used for synchronization between the flash memory driver and radio (:kconfig:option:`CONFIG_SOC_FLASH_NRF_RADIO_SYNC_MPSL`).
     This is done to work around the timeout in MPSL flash synchronization (``NCSDK-29354`` known issue).
     See :ref:`known_issues` for details.
+  * Note informing about :file:`dfu_application.zip` format version ``1`` support as part of :ref:`nrf_desktop_fwupd`.
+    The format version ``1`` is supported by the fwupd since the 1.9.25 release.
```

Comment:
This is added here: https://github.com/nrfconnect/sdk-nrf/compare/76008b67191ba20c34b28f21a13bb9af51106af0..6c9b139d2c84639ec90a3716717cbc681eb78c8a. This file could be removed.

Reply:
Removed the release note from this PR.